perun-core/src/main/java/cz/metacentrum/perun/core/impl/ServicesManagerImpl.java

```diff
@@ -339,6 +339,16 @@ public void unblockServiceOnDestination(int serviceId, int destinationId) {
 		}
 	}
 
+	@SuppressWarnings("ConstantConditions")
```

Comment:
This suppression seems to be redundant.

Reply:
Removed from this method. The suppression is used also on other methods, but I will leave that for possible cleanup later.